### ISSUE TITLE
No link previews in conversations with message timer

### ISSUE THREAD

**Issue by BobbyWibowo** [wireapp/wire-desktop#3253]
Wire version: 3.11.2912
Wire for web version: (not applicable)
Operating system: KDE neon 5.17.2 (Ubuntu 18.04.2) / 5.0.0-32-generic

What steps will reproduce the problem?

1. Preferences > Options > Create previews for links you send
2. Enable timed messages (I used 1 week in my tests)
3. Send any URLs with proper Open Graph tags
4. Wait a couple seconds for URL previews to appear on your/sender's side

What is the expected result?

URL previews appear on recipient's side

What is the actual result?

URL previews still won't appear on recipient's side even by then, and it seems like they never will

Please provide any additional information below.

If it's an actual intended behavior, I guess at least disables URL previews on the sender's side when the messages are on timer. But still enable it when timer is disabled, if the preference is also enabled.

**Comment by ffflorian** [comment]
Hi @BobbyWibowo, I can't reproduce this behavior. Timed messages should also generate link previews. Can you send me the link which fails to create a link preview?

**Comment by BobbyWibowo** [comment]
URL samples:
- https://safe.fiery.me
- https://xenforo.com/community/

<details>
<summary>Screenshot 1 (Wire Linux 3.11.2912)</summary>

![Screenshot_20191112_164700](https://user-images.githubusercontent.com/9364930/68660988-c355ac00-056c-11ea-8e2e-3281e4f5efc1.png)
> This was taken in Wire Linux 3.11.2912, from where the timed messages were sent.
> The first 2 messages were on timer (1 week).
> Their previews appeared properly on that device alone (sender's).
</details>

<details>
<summary>Screenshot 2 (Wire Android 3.41.848)</summary>

![Screenshot_20191112-164613](https://user-images.githubusercontent.com/9364930/68661074-f1d38700-056c-11ea-9c72-de5dd7447a1f.jpg)
> This was taken in Wire Android 3.41.848, same user account (sender's).
> Unable to see previews for the first 2 URLs, which were the ones on timer.
</details>

Basically, the same behavior as that Android screenshot will apply to my chat partner's devices (recipient's).
I forgot to confirm if they were having the same issue with their Wire Android, but they certainly had the same issue with their Wire Linux.

They've also tried sending the same URLs, and other URLs to me (basically them becoming sender's, and me becoming recipient's, instead).
And as expected, I couldn't see previews of their messages, but they could see their own (we've exchanged screenshots).

**Comment by ffflorian** [comment]
Ah I see what you mean. Good observation, I'll file a ticket!

**Comment by BobbyWibowo** [comment]
By the way, what does the title mean by "groups"? Cause I'm also having the issue in direct chat with another Wire user.
The screenshots were taken in a group with only me in it merely for screenshots for this GitHub issue.

**Comment by ffflorian** [comment]
Thanks for the hint @BobbyWibowo, I updated the title.

**Comment by ffflorian** [comment]
@BobbyWibowo this should be fixed by now.
